libminifi/test/expression-language-tests/ExpressionLanguageTests.cpp

```diff
@@ -203,6 +209,12 @@ TEST_CASE("GetFile PutFile dynamic attribute", "[expressionLanguageTestGetFilePu
       get_file,
       processors::GetFile::KeepSourceFile.getName(),
       "false");
+  auto update = plan->addProcessor(
+      "UpdateAttribute",
+      "UpdateAttribute",
+      core::Relationship("success", "description"),
+      true);
+  update->setDynamicProperty("prop_attr", "${'nifi.my.own.property'}_added");
```

Comment:
@jdye64 is this what you need?

Reply:
Yes it is. However I just want to make sure we document that those global properties must start with "nifi." could we add that to the documentation somewhere?

Reply:
Sure, that's planned. Just a POC yet to make sure it's what we need. 
Maybe @phrocker has some better idea to access global configuration here.